runtimes/swift-5.5/Sources/App/routes.swift

```diff
@@ -110,19 +100,51 @@ func routes(_ app: Application) throws {
             return RequestResponse.unauthorized()
         }
 
+        setvbuf(stdout, nil, _IONBF, 0)
+        setvbuf(stderr, nil, _IONBF, 0)
+
+        let outPipe = Pipe()
+        let errPipe = Pipe()
+        var outString = ""
+        var errString = ""
+
```

Comment:
Since console logs are not working in swift, can we revert these changes in swift to simply return an empty string for `stdout`?

Reply:
Well I'm okay either way. as it's not working

Reply:
@abnegate please address this when you get the time.